packages/components/src/combobox/options-dropdown/options-dropdown.tsx

```diff
@@ -55,7 +55,18 @@ function isValueChecked(value: string | string[] | undefined | null, optionValue
 function Option({ data, withCheckIcon, value, checkIconPosition, unstyled, renderOption }: OptionProps) {
 	if (!isOptionsGroup(data)) {
 		const checked = isValueChecked(value, data.value);
-		const check = withCheckIcon && checked && <CheckIcon className={classes.optionsDropdownCheckIcon} />;
+		const check = withCheckIcon && (
+			<Checkbox
```

Comment:
This will impact the rest of the combobox components, only MultiSelect is rendered with Checkbox. This is already done in Prism Portal project (see extended MultiSelect from theme package).